components/card/card.tsx

```diff
@@ -16,6 +17,7 @@ type Props = {
 
 export default class Card extends React.PureComponent<Props> {
     public static Header = CardHeader;
+    public static HeaderContent = CardHeaderContent;
```

Comment:
0/5, but not sure if this needs to be a member of the card if it's only used by the admin console. It can be a separate component.

Reply:
@devinbinnie 
I actually recommended him this only because I got inspiration from how the `Menu` component is done
![image](https://user-images.githubusercontent.com/17804942/107406036-f66f4e00-6ad5-11eb-844e-aed693297d17.png)
I would still say keep it this way only because if it's a totally separate component, we will probably be passing it as a child of Card.Header almost always anyways so might as well be a part of Card. Although currently we only use it in admin console, it would be easier to extend outside if ever needed.